src/ResourceManager/Common/Commands.ResourceManager.Common/Commands.ResourceManager.Common.csproj

```diff
@@ -11,8 +11,6 @@
     <AssemblyName>Microsoft.Azure.Commands.ResourceManager.Common</AssemblyName>
```

Comment:
Looks like there are some unintended changes. Can you please revert changes to these files?